models/models/markov_chain/markov_chain.py

```diff
@@ -7,7 +7,9 @@
 from midi.decode import get_array_of_notes
 from midi.encode import get_file_from_standard_features
 
-from models.music_model import MusicModel, ProgressCallback, ProgressMetadata
+import time
+
+from models.music_model import MusicModel, ProgressCallback, ProgressMetadata, SeriesProgress
```

Comment:
Could you rebase please? Something went wrong with commits it seems, this is already on main